src/Serilog.Sinks.Logz.Io/Serilog.Sinks.Logz.Io.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net45</TargetFrameworks>
```

Comment:
To be honest, I do not like mixed changes in one pull request. Any reason, why net45 was removed from here?

Reply:
Also .netstandard1.4 is not compatible with .NET 4.5, so the change would mean - the projects which are targeting less than .NET 4.6.1 will be forced to update. Please revert this change.

Reply:
I had changed it because my test runner complained (something with `net45` being not supported anymore). But I've reverted it and ran tests in another way.